tracer/test/snapshots/WcfTests.__metadataSchemaVersion=v0_binding=BasicHttpBinding_enableNewWcfInstrumentation=False_enableWcfObfuscation=False.verified.txt

```diff
@@ -17,7 +17,8 @@
       runtime-id: Guid_1,
       span.kind: server,
       version: 1.0.0,
-      _dd.p.dm: -0
+      _dd.p.dm: -0,
+      _dd.p.tid: 1234567890abcdef
```

Comment:
I've probably missed something along the way, but what is this?!

Reply:
Trace Id introduced with 128 bits